.github/CONTRIBUTING.rst

```diff
@@ -8,17 +8,17 @@ Build Status
 
   + `Stable buildbots <http://buildbot.python.org/3.x.stable/>`_
 
-- 3.7
+- 3.9
 
-  + `Stable buildbots <http://buildbot.python.org/3.7.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/3.9.stable/>`_
 
-- 3.6
+- 3.8
 
-  + `Stable buildbots <http://buildbot.python.org/3.6.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/3.8.stable/>`_
 
-- 2.7
+- 3.7
 
-  + `Stable buildbots <http://buildbot.python.org/2.7.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/3.7.stable/>`_
```

Comment:
I wonder if we should simplify this to just link to the 3.x stable set and mention that you can adjust the URL manually (replace the `x` with the minor version of your choice) to get the stable set for a maintenance branch.

Reply:
Good idea : )
I will take a look(I am sure we can do it or not now)